### ISSUE TITLE
Error when user tries to drop text

### ISSUE THREAD

**Issue by crizant-mpower** [deakjahn/flutter_dropzone#68]
Select some text on the address bar, then drag it over the dropzone,
the following error is thrown:
```
Error: Expected a value of type
'File$', but got one of type 'String'
../dart-sdk/lib/_internal/js_dev_runtim
e/private/ddc_runtime/errors.dart
288:49      throw_
../dart-sdk/lib/_internal/js_dev_runtim
e/private/ddc_runtime/errors.dart 121:3
castError
```

**Comment by deakjahn** [comment]
@crizant-mpower Yep but I'm not yet sure what the best solution would be. `onDrop()` has a `File` as second argument. We could change it to `dynamic` (just like `onDropMultiple()`) but that would be a breaking change of the interface. We could also add a different `onDrop()` variant but I can't really see why.

**Comment by deakjahn** [comment]
Wait a minute... The handler itself is OK, it was an intermediate function somewhere with the `File` argument...

Check out the changes, please.

**Comment by crizant-mpower** [comment]
I'm fine with the changes. Looking forward to the next release.

**Comment by deakjahn** [comment]
Published now.

**Comment by crizant-mpower** [comment]
Sorry, it seems you forgot to update the `onDropMultiple` method.

**Comment by deakjahn** [comment]
But you can't drop more than one string at a time, can you?

**Comment by crizant-mpower** [comment]
> @crizant-mpower OK, this is definitively something I don't want to publish before you test it. :-) See [7230085](https://github.com/deakjahn/flutter_dropzone/commit/72300858eb32c865fbf7b2f10624dda5a82791a1)
> 
> Basically, getting the string is async so we actually went on to fire (or not fire) `onDropMultiple` before the string got a chance to be set (files are not problematic because, for some obscure reason, getting them is sync). I added a private helper function to make getting the string synchronous. The change is straightforward and seems to work for my simple tests but let's be sure first...

I tested and it works like a charm. 🎉

**Comment by deakjahn** [comment]
Then it's released again, thanks.
